pkg/processor/reconciler_helper.go

```diff
@@ -153,7 +153,7 @@ func (h *ReconcilerHelper) PruneBalances(
 	currency *types.Currency,
 	index int64,
 ) error {
-	if h.config.Data.PruningDisabled {
+	if h.config.Data.PruningDisabled || h.config.Data.PruningBalanceDisabled {
```

Comment:
We can fully decouple pruning, so only `h.config.Data.PruningBalanceDisabled` is needed here

Reply:
I fixed it that way to respect the intent of the original author.
Okay, I will fix the code so that only `PruningBalanceDisabled` is needed. I also think that is better.